compiler/luci-value-test/requirements.txt

```diff
@@ -1,2 +1,2 @@
-tensorflow==1.13.1
+tensorflow==2.2.0
```

Comment:
This change seems not relevant to this PR.

Reply:
Oh, sorry, I committed this accidently. I will revert it

Reply:
Now I saw the above comment. If tf version is backward compatible, it's ok to change the version.

Reply:
I don't think this is about backward compatibility. Please remove this change.
And this file is different from other files.